pkg/lagoon/users/main.go

```diff
@@ -106,7 +106,7 @@ func (ud *Data) AddItem(userData UserData) {
 func distinctObjects(objs []UserData) (distinctedObjs []UserData) {
```

Comment:
I think that given we're cleaning stuff up, this function (and the next) can probably be made tighter with the new slices library.

Maybe something like
```
func distinctObjects(objs []UserData) (distinctedObjs []UserData) {
	var output []UserData

	ids := []string{}

	for _, e := range objs {
		if !slices.Contains(ids, e.ID) {
			ids = append(ids, e.ID)
			output = append(output, e)
		}
	}

	return output
}
```

Reply:
Ahh nice. I didn't go too deep into a lot of the code itself more formatting. But this seems simple enough to slide in

Reply:
Updated